packages/terra-clinical-action-header/package.json

```diff
@@ -23,29 +23,29 @@
   },
   "homepage": "https://github.com/cerner/terra-clinical#readme",
   "devDependencies": {
-    "terra-toolkit": "^0.x"
+    "terra-toolkit": "^0.3.1"
   },
   "peerDependencies": {
-    "react": "15.4.2",
-    "react-dom": "15.4.2",
-    "terra-base": "^0.x",
-    "terra-button": "^0.x",
-    "terra-button-group": "^0.x",
-    "terra-icon": "0.x",
-    "terra-mixins": "^1.0.0",
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^0.4.1",
+    "terra-button": "^0.4.1",
+    "terra-button-group": "^0.2.1",
+    "terra-icon": "^0.4.1",
+    "terra-mixins": "^1.1.1",
     "terra-clinical-header": "^0.x",
-    "terra-responsive-element": "^0.x"
+    "terra-responsive-element": "^0.3.1"
```

Comment:
If you want to set a minimum version you could do something like this:
```
0.x > 0.3.1
```
![screen shot 2017-05-17 at 4 05 10 pm](https://cloud.githubusercontent.com/assets/8146159/26176118/affc1f0a-3b1a-11e7-853f-437c6db62920.png)